storage-inventory/ops/README.md

```diff
@@ -8,20 +8,17 @@
 A representation of a file and its metadata in the SI database, specifically the _inventory.Artifact_ table (see the [SI data model](https://github.com/opencadc/storage-inventory/tree/master/storage-inventory-dm)).  The term _artifact_ is often used to refer to both the database representation of a file and the file itself as one thing, but when an SI application is acting on an _artifact_, it specifically refers to the database.
 - ➡️ an artifact _URI_ is a unique idenitfier for a file (or object) stored in SI.  The last path component of a _URI_ is always the _filename_.
 
-<a id='term-bucket'>**bucket**</a>  
-SI can organize units of work (number of files or artifacts) by `buckets` and applications can be configured to work on subsets of these buckets.  
+<a id='term-bucket'>**bucket**</a> 
+SI assigns a `bucket` to files on storage and to artifacts in the inventory database.  The bucket for an artifact and the bucket for the file described by that artifact are not necessarily the same. 
```

Comment:
I'm afraid this is not clear either. Is the storage bucket exposed anywhere (config files) or just a configuration of the S3/Swift system?